### ISSUE TITLE
This is the greatest computer program in the world.

### ISSUE THREAD

**Issue by micha** [mpereira/tty-solitaire#2]
I love it dude. Thank you! Works great!

**Comment by Tethtibis** [comment]
@mpereira 

    First off, this is amazingly well done, and a great game. Thank you for this. I actually have it installed on my cloud server with screen, so I can play it from anywhere and detach the screen to continue the game later.

    Second, is there a way to give infinite passes? I've tried starting it with `ttysolitaire -p=99` , but it only goes through the deck once. I've tried multiple smaller numbers, but it seems when you specify a number of passes, it takes it to 0 passes, regardless.

**Comment by mpereira** [comment]
@Tethtibis 

> First off, this is amazingly well done, and a great game. Thank you for this. I actually have it installed on my cloud server with screen, so I can play it from anywhere and detach the screen to continue the game later.

Thanks, I'm glad you like it!

> Second, is there a way to give infinite passes? I've tried starting it with "ttysolitaire -p=99", but it only goes through the deck once. I've tried multiple smaller numbers, but it seems when you specify a number of passes, it takes it to 0 passes, regardless.

I get the same behavior here. Apparently `getopt_long` is not doing the right thing when it sees an equals sign in between flags and values. Try `ttysolitaire -p 99`, without the equals sign, it seems to do the right thing.

**Comment by Tethtibis** [comment]
@mpereira 

> I get the same behavior here. Apparently the option parsing code is not doing the right thing when it sees an equals sign in between short flags and values. Try ttysolitaire -p 99, without the equals sign (or even ttysolitaire --passes=99), it seems to do the right thing.

That absolutely worked. thank you. :O)

**Comment by greno4ka** [comment]
It's really great!!!

**Comment by mpereira** [comment]
@greno4ka 

> It's really great!!!

Thanks for taking the time to say that. Enjoy!

**Comment by rullinoiz** [comment]
yes this is epic. found it on brew and decided to check it out. no regrets 10/10

**Comment by dther** [comment]
Resurrecting this issue for the sole purpose of saying that I loved stumbling upon this on the Arch repositories ;)

**Comment by mpereira** [comment]
Thanks for letting me know, I appreciate it :)
